### DISCUSSION TITLE
nextjs and prisma

### DISCUSSION THREAD

**Discussion by oscarayala431** [vercel/next.js#21531] [Help]
I am starting with nextjs and to keep practicing I wanted to make a login and validate a user and password, I just discovered also the use of prisma to make queries to my mysql database, my question is after validating in the database how to save that in a session, since the nextjs documentation says that when we want to do something like that it should be a provider independent solution for authentication. Because what I had in mind was to save some user data using next-iron-session but I don't know if I'm right. I could use some feedback. Thanks for the help.

**Comment by leerob** [comment]
Check out next-auth + Prisma: https://next-auth.js.org/schemas/adapters#prisma-adapter
